services/cron.ts

```diff
@@ -0,0 +1,32 @@
+const cron = require('node-cron');
+const Routine = require('../models/routine.model');
+
+// run a cron job every minute
+export const checkEveryMinute= cron.schedule('* * * * *', async () => {
+    console.log('running a task every minute');
+
+    // Since I did not know about the time format I return medicines between the start and end time
+    const drugsToBeNotified = await Routine.aggregate([
+        {
+            "$match": {
+                //matching the current time with the start and end time of the medicine
+                "medicineStartTime": {
+                    "$lte": new Date().getTime()
+                },
+                "medicineEndTime": {
+                    "$gte": new Date().getTime()
```

Comment:
These are not the correct fields.. add the time in the model. Create a new issue if you have to.